Src/BootCamp.Chapter/BinaryConverter.cs

```diff
@@ -6,14 +6,45 @@ namespace BootCamp.Chapter
 {
     public static class BinaryConverter
     {
+        private const int OnBit = 1;
+        private const int OffBit = 0;
+
         public static long ToInteger(string binary)
         {
-            return 0;
+            var result = 0; // We assume that the resulting number will be 0
```

Comment:
Why would we make such an assumption? I think a better comment would be...
```
Default number is 0.
```
But I would remove the comment altogether, because it's a pretty standard thing to do it like this.